lib/src/layer.dart

```diff
@@ -67,9 +70,54 @@ class Layer {
         tileIndex += 4;
 
         // Read out the flags
-        var flipped_horizontally = (globalTileId & FLIPPED_HORIZONTALLY_FLAG);
-        var flipped_vertically = (globalTileId & FLIPPED_VERTICALLY_FLAG);
-        var flipped_diagonally = (globalTileId & FLIPPED_DIAGONALLY_FLAG);
+        bool flippedHorizontally = (globalTileId & FLIPPED_HORIZONTALLY_FLAG) == FLIPPED_HORIZONTALLY_FLAG;
+        bool flippedVertically = (globalTileId & FLIPPED_VERTICALLY_FLAG) == FLIPPED_VERTICALLY_FLAG;
+        bool flippedDiagonally = (globalTileId & FLIPPED_DIAGONALLY_FLAG) == FLIPPED_DIAGONALLY_FLAG;
+
+        //Translating the flags into rotation
+        if(
+          !flippedHorizontally && 
+          flippedVertically && 
+          flippedDiagonally){
+            tileRotation[y][x] = 1; //90 °
+        }else if(
```

Comment:
this is not formatted properly, please do:
```
    if (...) {
```

Reply:
we can have a method that takes in `globalTileId` and returns the rotation int (1, 2, etc). and here you just do
```
tileRotation[y][x] = foo(globalTileId)
```